.github/workflows/ci-dockers.yml

```diff
@@ -70,12 +71,11 @@ jobs:
         include:
           # These are the base images for PL release docker images,
           # so include at least all of the combinations in release-dockers.yml.
-          - {python_version: "3.9", pytorch_version: "1.11", cuda_version: "11.3.1"}
           - {python_version: "3.9", pytorch_version: "1.12", cuda_version: "11.7.1"}
+          - {python_version: "3.9", pytorch_version: "1.13", cuda_version: "11.8.0"}
           - {python_version: "3.9", pytorch_version: "1.13", cuda_version: "12.0.1"}
+          - {python_version: "3.10", pytorch_version: "2.0", cuda_version: "11.8.0"}
           - {python_version: "3.10", pytorch_version: "2.0", cuda_version: "12.0.1"}
-          # these are used in Azure GPU CI
-          - {python_version: "3.10", pytorch_version: "2.0", cuda_version: "11.7.1"}
```

Comment:
Benchmarks is still using this one. If kept, then it shouldn't be removed from this config

Reply:
Let's bump it too